src/core/Spec.js

```diff
@@ -14,6 +14,9 @@ getJasmineRequireObj().Spec = function(j$) {
     this.queueRunnerFactory = attrs.queueRunnerFactory || function() {};
     this.catchingExceptions = attrs.catchingExceptions || function() { return true; };
     this.throwOnExpectationFailure = !!attrs.throwOnExpectationFailure;
+    this.timer = attrs.timer || { start: function(){}, elapsed: function(){ return null; }
```

Comment:
I'd like to see this share the [noopTimer from HtmlReporter](https://github.com/jasmine/jasmine/blob/master/src/html/HtmlReporter.js#L3). Probably Suite as well.